ironfish/src/primitives/rawTransaction.ts

```diff
@@ -53,17 +57,54 @@ export class RawTransaction {
     >
   }[] = []
 
-  size(): number {
+  size(senderPublicAddress: string): number {
```

Comment:
What do you think about renaming this? I feel like `.size()` sets a specific expectation. Maybe `postedSize()`, I don't know. I don't feel super strongly here